app/locales/ja/translations.ts

```diff
@@ -33,7 +33,7 @@ export default {
         filter: 'フィルター',
         revert: '復帰',
         save: '保存',
-        ellipsis: '\u2026',
+        ellipses: '\u2026',
```

Comment:
This key was renamed to the singular: `ellipsis`

Reply:
We'll need this updated to pass the tests